examples/tools/autotools/autotools_toolchain/build_project_autotools_toolchain.rst

```diff
@@ -0,0 +1,215 @@
+.. _examples_tools_autotools_autotools_toolchain_build_project_autotools_toolchain:
+
+Build a simple Autotools project using Conan
+============================================
+
+In this example, we are going to create a string formatter application
+that uses one of the most popular C++ libraries: `fmt <https://fmt.dev/latest/index.html/>`__.
+
+We'll use Autotools as build system and pkg-config as helper tool in this case, so you should get them installed
+on Linux and Mac before going forward with this example. In case you do not have them installed already, you can
+`Conan Center <https://conan.io/center/>`_ and install them as build requirements.
```

Comment:
I would remove the sentence about installing them as build requirements, because if this tutorial is example is for newbies it may be a bit confusing, the most typical thing is to have that installed on the system.

Reply:
Done.